README.md

```diff
@@ -1 +1,38 @@
-# Awesome-books-Modular-Javascript
+## Awesome books with Modular Javascript
+
+![screenshot](https://raw.githubusercontent.com/amitmerchant1990/electron-markdownify/master/app/img/markdownify.gif)
+
+## How To Use
+
+To clone and run this application, you'll need [Git](https://git-scm.com) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer. From your command line:
+
+```bash
+# Clone this repository
+$ git clone https://github.com/amitmerchant1990/electron-markdownify
+
+# Go into the repository
+$ cd electron-markdownify
+
+# Install dependencies
+$ npm install
+
+# Run the app
+$ npm start
+```
+## Credits
+
+This project uses the following open source packages:
+
+- HTML
+- CSS
+- Modular JavaScript
+- Luxon
+
+## License
+
+MIT
+
+---
+
+> GitHub [@AkshithaReddy1899](https://github.com) &nbsp;&middot;&nbsp;
+> Twitter [@Akshitha Reddy](https://twitter.com)
```

Comment:
- [ ] Properly document your work following the [Microverse template](https://github.com/microverseinc/readme-template). This helps you practise documentation as well as your communication (written) skills.